common/build/eslint-plugin-fluid/src/test/example/no-unchecked-record-access/nullableIndexedRecord.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/*
+ * Nullable index signature
+ */
+
+/* Constants and Variables */
+type NullableIndexSignatureType = { [key: string]: string | null };
+const nullableIndexedRecord: NullableIndexSignatureType = { a: "hello", b: null };
+
+/* Conditional Checks */
+if (nullableIndexedRecord.a) {
+	nullableIndexedRecord.a.length; // ok: Within a truthy check, 'a' is guaranteed to be defined
+}
+
+/* Function Calls */
+function recordAFnExpectsStringOrNull(record: NullableIndexSignatureType): string | null {
+	return record.a; // ok: Returning index property 'a' to string or null variable, 'a' might not be present
+}
+
+AFnExpectsStringOrNull(nullableIndexedRecord.a); // ok: Passing index property 'a' to a function that accepts null is fine
+
+/*
+ * Variable Assignments
+ */
+
+const aExpectingStringOrNull: string | null = nullableIndexedRecord.a; // ok: Assigning index property 'a' to string or null variable, 'a' might not be present
```

Comment:
This is not okay. The comment says it might not be present. How could it be okay?